ios/Classes/RoktMethodCallHandler.swift

```diff
@@ -106,6 +111,21 @@ struct RoktMethodCallHandler {
         }
     }
     
+    private func registerPartnerFonts(_ typefaces: Dictionary<String, String>) {
+        let bundle = Bundle.main
+        for (_, fileName) in typefaces {
+            let fontKey = registrar.lookupKey(forAsset: fileName)
+            let path = bundle.path(forResource: fontKey, ofType: nil)
+            let fontData = NSData(contentsOfFile: path ?? "")
+            let dataProvider = CGDataProvider(data: fontData!)
+            let fontRef = CGFont(dataProvider!)
+            var errorRef: Unmanaged<CFError>? = nil
+            if let fr = fontRef {
+                CTFontManagerRegisterGraphicsFont(fr, &errorRef)
+            }
+        }
+    }
```

Comment:
@danial-rokt Please have a look when you get a chance

Reply:
Thanks for doing this,
We have moved from  CTFontManagerRegisterGraphicsFont  to CTFontManagerRegisterFontsForURL
https://github.com/ROKT/sdk-ios-source/blob/release-4.1.x/Rokt-Widget/Classes/Shared/Util/FontManager.swift#L169
As we have the path, I think it is easier and more reliable to register with the URL.

Reply:
Thanks. Updated.
The first time it is success and the error is `nil`. Second time it is returning an error code `105`, `CTFontManagerError.alreadyRegistered` which is expected I guess.
```
Optional(Swift.Unmanaged<__C.CFErrorRef>(_value: Error Domain=com.apple.CoreText.CTFontManagerErrorDomain Code=105 "Font registration was unsuccessful." UserInfo={NSLocalizedDescription=Font registration was unsuccessful., CTFontManagerErrorFontURLs=(
    "file:///Users/thomson/Library/Developer/CoreSimulator/Devices/74528584-E110-4498-A419-91D5B5DECACD/data/Containers/Bundle/Application/43746A1B-5E48-4FB6-83D3-1A9F0106F40C/Runner.app/Frameworks/App.framework/flutter_assets/fonts/NunitoSans-Bold.ttf"
)}))
```

Reply:
Thanks, Yes it is expected.
The font registration will fail if it is already registered.